sysid-application/src/integrationtest-generation/native/cpp/GenerationTest.cpp

```diff
@@ -145,13 +135,13 @@ TEST_F(GenerationTest, Drivetrain) {
   // Encoders + Motor controllers already tested, now just need to test the
   // gyros
   for (auto&& gyro : sysid::kGyros) {
-    if (wpi::StringRef(gyro) == "Romi") {
+    if (std::string_view{gyro} == "Romi") {
```

Comment:
We could probably also skip the NavX until we actually get the binaries for it since it seems to crash on the [NavX](https://github.com/wpilibsuite/sysid/pull/144/checks?check_run_id=2702756388).

Reply:
The original GenerationTest.cpp doesn't crash though.

Reply:
That's true, but it does seem like the NavX test case is bad.